### PR TITLE
Add telemetry to reqs + add file for e2e tests

### PR DESCRIPTION
**Pull request by MerelTheisenQB**
## Motivation and Context
Kedro telemetry has been released and now the e2e tests were failing for Kedro-Docker. Added a `.telemetry` file for the e2e tests in the same way as we do in starters.

## How has this been tested?
All tests pass now.

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [X] Assigned myself to the PR
- [ ] Added tests to cover my changes
- [ ] Added new entries to the `RELEASE.md` file

## Notice

- [X] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

**Review by AntonyMilneQB — APPROVED** [review]
👍

File: features/steps/cli_steps.py [hunk 1]
```diff
@@ -307,7 +310,7 @@ def check_jupyter_nb_proc_on_port(context: behave.runner.Context, port: int):
         print_error=False,
         context=context,
         url=url,
-        string="Jupyter Notebook",
+        string="Jupyter",
```
Comment (MerelTheisenQB) [inline features/steps/cli_steps.py:313]: When running jupyter lab, the word `notebook` doesn't appear in the html.

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
+kedro-telemetry~=0.1.0
```
Comment (lorenabalan) [inline requirements.txt:1]: Mm I don't think we should force users to install telemetry in order to be able to use a plugin, it should be an optional thing that the user can do without in their environment.

**Review by lorenabalan — APPROVED** [review]

**Review by ignacioparicio — APPROVED** [review]
LGTM!

**Pull request merged**